### PR TITLE
Implement status filtering from the statistics table

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

File: ara/views.py [hunk 1]
```diff
@@ -74,17 +74,30 @@ def playbook(playbook):
 
 
 @app.route('/playbook/<playbook>/host/<host>')
-def playbook_host(playbook, host):
+@app.route('/playbook/<playbook>/host/<host>/status/<status>')
```
Comment (larsks) [inline ara/views.py]: I think that we should probably be using query parameters for filtering, as probably a more flexible and REST-ish type of solution.  E.g., `/playbook/playbookid?status=ok&host=localhost`.  This opens up the possibilities for more complex filters (`?status=ok&status=changed`).

File: ara/templates/playbook.html [hunk 2]
```diff
@@ -39,11 +39,11 @@
     {% for stat in playbook.stats %}
     <tr>
       <td>{{ make_link('playbook_host', stat.host.name, playbook=playbook.id, host=stat.host.name) }}</td>
-      <td>{{ stat.ok }}</td>
-      <td>{{ stat.changed }}</td>
-      <td>{{ stat.failed }}</td>
-      <td>{{ stat.skipped }}</td>
-      <td>{{ stat.unreachable }}</td>
+      <td>{% if stat.ok >= 1 %}{{ make_link('playbook_host', stat.ok, playbook=playbook.id, host=stat.host.name, status='ok') }}{% else %}{{ stat.ok }}{% endif %}</td>
```
Comment (larsks) [inline ara/templates/playbook.html]: Too much boilerplate code!  If nothing else we should use a jinja2 macro for this.

File: ara/views.py (hunk 1 above)
Comment (larsks) [inline ara/views.py]: But don't try to fix that now.  I've submitted #33 as a suggested modification of this PR; it's mostly just a reduction in code through the use of macros.  I haven't attempted any filtering changes.

**Comment by dmsimard** [comment]
Merged https://github.com/dmsimard/ara/pull/33 instead

**Pull request closed without merging**